tests/testthat/testdata/create_exp_score_fixtures.R

```diff
@@ -0,0 +1,56 @@
+library(testthat)
+library(hubData)
+library(hubEvals)
+library(dplyr)
+
```

Comment:
Not needed since you are declaring them using the `package::function` syntax. These declarations will end up poisoning your environment. 

```suggestion
```